tests/unit/database/test_image.py

```diff
@@ -89,6 +91,10 @@ def runTest(self):
                                                                'project 1')
         self.assertEqual(img_id, 2)
 
+        with pytest.raises(db_exceptions.ImageNotFoundException):
+            img_id = self.db.image.fetch_id_with_name_from_project(
+                'some_', 'project 1')
```

Comment:
Also when I initially raised my tests PR the tests here were not split to keep the number of classes small. Since we now agreed to use functions in the test case class it would be better to write each test/assert as a separate function.

It will be better if you can split all the tests in this particular class or atleast your test.